Util.js

```diff
@@ -12,6 +12,10 @@ function isInt_ (n) {
   return n % 1 === 0
 }
 
+function IsNumeric_ (val) {
```

Comment:
Is there a reason you capitalized this function's name?

Reply:
Copy pasta oversight.